data-esnext.js

```diff
@@ -2206,7 +2206,7 @@ exports.tests = [
   }
 },
 {
-  name: 'getting last item from array',
+  name: 'Getting last element from array',
```

Comment:
In the case of `.lastIndex`, I'm don't think that "element" better.

Reply:
I think it would be better just revert it for consistency with the header of the proposal repo.

Reply:
I copied the title from here: https://github.com/tc39/proposals/blob/master/stage-1-proposals.md
Fixed.